data/companies.json

```
@@ -176,6 +176,22 @@
         "youtube": "https://www.youtube.com/c/CloudfactoryHQ"
       }
     },
+    {
+        "name": "Clusus Information Solutions",
+        "establishedAt": 1535414400,
+        "location": "Jhamsikhel, Lalitpur",
+        "website": "https://www.clusus.com/",
+        "email": "Info@clusus.com",
```

Maybe lowercase on email helps.
```suggestion
        "email": "info@clusus.com",
```